src/frontend/Player/createPlayer.ts

```diff
@@ -20,7 +20,40 @@ export const createPlayer: VideoPlayerCreator = (
         locale,
         onReady,
       );
+
+      const trackTextKind: { [key in timedTextMode]?: string } = {
+        [timedTextMode.CLOSED_CAPTIONING]: 'captions',
+        [timedTextMode.SUBTITLE]: 'subtitles',
+      };
+
+      const addTrackToPlayer = (
+        track: TimedText,
+        languages: { [key: string]: string },
+      ) => {
+        player.addRemoteTextTrack(
+          {
+            id: track.id,
+            src: track.url,
+            language: track.language,
+            kind: trackTextKind[track.mode] as 'captions' | 'subtitles',
+            label: languages[track.language] || track.language,
+          },
+          true,
+        );
+      };
+
+      const removeTrackFromPlayer = (track: TimedText) => {
+        if (track.id) {
+          player.removeRemoteTextTrack(
+            player.remoteTextTracks().getTrackById(track.id) as any,
```

Comment:
unfortunately the videojs type is incomplete. There is a `remoteTextTracksEls` methods thant contains what you need without using `as any`. But there is lot of interfaces to add to fill the gap... I think we have no other solution than keeping that...